### PR TITLE
[WIP] Analysis and report generation.

### PR DESCRIPTION
**Pull request by dfhahn**
## Description
This PR addresses #13 and is based on Victoria Lim's [benchmarkff](https://github.com/MobleyLab/benchmarkff)

## Todos
Notable points that this PR has either accomplished or will accomplish.
  - [ ] Read in minimized / computed conformers and their energy values.
  - [ ] Reference the energies to lowest QM structure
  - [ ] Calculate Torsion Fingerprints, RMSD and relative conformer energy differences (ddE) with respect to QM (ref) structures
  - [ ] Plot results and generate a report. 

## Questions
- [ ] Question1

## Status
- [ ] Ready to go

### CODE REVIEW

File: openff/benchmark/analysis/draw.py [hunk 1]
```diff
@@ -0,0 +1,519 @@
+def draw_scatter(
```
Comment (dotsdl) [inline openff/benchmark/analysis/draw.py:1]: Need to add imports yet.

**Review by dotsdl — APPROVED** [review]
Great work @dfhahn! Just a few comments from walking through the analysis component implementation, but otherwise feel free to merge so we can begin consolidating e.g. `readwrite`. Thank you for doing this!

File: openff/benchmark/analysis/analysis.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    ref_confs = {}
+    dataframe.loc[:,'ref_conf'] = False
+    for mid in dataframe.molecule_index.unique():
+        confs = dataframe.loc[dataframe.molecule_index==mid]
+        if confs.shape[0] == 1:
+            ref_conf = confs.name[0]
+        else:
+            ref_conf = confs.final_energy.idxmin()
+        ref_confs[mid] = ref_conf
+        dataframe.loc[ref_conf, 'ref_conf'] = True
```
[the remark this answers is not in the document]
Comment (dfhahn) [reply openff/benchmark/analysis/analysis.py:34]: Thanks!

File: openff/benchmark/analysis/metrics.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+"""
+metrics.py
+
+Metrics calculation for the analysis/report part of the openff-benchmark workflow
+
+By:      David F. Hahn
+Version: Nov 25 2020
+
+"""
+
```
[the remark this answers is not in the document]
Comment (dfhahn) [reply openff/benchmark/analysis/metrics.py:12]: Resolved by d0ccaa7830fd5c6b82d2068ae762c9c49c1813d2

File: openff/benchmark/analysis/draw.py (hunk 1 above)
Comment (dfhahn) [reply openff/benchmark/analysis/draw.py:22]: Added in 6289f70

**Pull request merged**